_posts/2023-08-23-Announcing-Data-Prepper-2.4.0.md

```diff
@@ -0,0 +1,286 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.4.0"
+authors:
+- sudipto
+- kkondaka
+- nsifmoh
+- dlv
+date: 2023-08-23 11:30:00 -0500
+categories:
+  - releases
+meta_keywords: Data Prepper 2.4.0
```

Comment:
Suggested meta update: 

**meta_keywords:** Data Prepper, Apache Kafka, Amazon S3 bucket, OpenSearch ingestion

**meta_description:** OpenSearch has launched Data Prepper 2.4.0 with feature support for Apache Kafka source, S3 batch processing, filtering in sinks, S3 sink codecs, and high-cardinality anomaly detection.